cmd/stringer/golden_test.go

```diff
@@ -397,6 +401,80 @@ func (i Token) String() string {
 }
 `
 
+const toLower_in = `type Day int
+const (
+	Monday Day = iota
+	Tuesday
+	Wednesday
+	Thursday
+	Friday
+	Saturday
+	Sunday
+)
+`
+
+const toLower_out = `func _() {
+	// An "invalid array index" compiler error signifies that the constant values have changed.
+	// Re-run the stringer command to generate them again.
+	var x [1]struct{}
+	_ = x[Monday-0]
+	_ = x[Tuesday-1]
+	_ = x[Wednesday-2]
+	_ = x[Thursday-3]
+	_ = x[Friday-4]
+	_ = x[Saturday-5]
+	_ = x[Sunday-6]
+}
+
+const _Day_name = "mondaytuesdaywednesdaythursdayfridaysaturdaysunday"
+
+var _Day_index = [...]uint8{0, 6, 13, 22, 30, 36, 44, 50}
+
+func (i Day) String() string {
+	if i < 0 || i >= Day(len(_Day_index)-1) {
+		return "Day(" + strconv.FormatInt(int64(i), 10) + ")"
+	}
+	return _Day_name[_Day_index[i]:_Day_index[i+1]]
+}
+`
+
+const toUpper_in = `type Day int
```

Comment:
warning: don't use underscores in Go names; const toUpper_in should be toUpperIn (golint)